src/types/api.d.ts

```diff
@@ -0,0 +1,45 @@
+export type AssetInfo =
```

Comment:
whatabout using `AssetInfo`, `Pair`, `PoolAsset` in `pair.d.ts`?

Reply:
I believe it would be beneficial to separate the interface definition of the API response from the interface definition of the contract response. 
However, I am open to any other suggestions you may have.

Reply:
I was so sure that api interface will try to be compatible with the contract's one tho, let me leave it @jbamlee's decision

Reply:
I agree with @honeymaro. Contract and API messages can differ.

I used the same responses to minimize changes to the web app and avoid deployments. I apologize for confusing you @jhlee-young